SVFG: compilation error due to null pointer

```
SVF/lib/Graphs/SVFG.cpp: In member function ‘const SVF::SVFFunction* SVF::SVFG::isFunEntrySVFGNode(const SVFGNode*) const’:
SVF/lib/Graphs/SVFG.cpp:716:31: error: ‘this’ pointer is null [-Werror=nonnull]
  716 |             return phi->getFun();
      |                    ~~~~~~~~~~~^~
cc1plus: all warnings being treated as errors

```
`phi` at that line is NULL due to the if-else chain (L 704). I guess you mean `mphi` here? 
https://github.com/SVF-tools/SVF/blob/09833590e6c87b1e00185b96e52d2b8582c93610/lib/Graphs/SVFG.cpp#L716

--- a/lib/Graphs/SVFG.cpp
+++ b/lib/Graphs/SVFG.cpp
@@ -713,7 +713,7 @@ const SVFFunction* SVFG::isFunEntrySVFGNode(const SVFGNode* node) const
     else if(const InterMSSAPHISVFGNode* mphi = SVFUtil::dyn_cast<InterMSSAPHISVFGNode>(node))
     {
         if(mphi->isFormalINPHI())
-            return phi->getFun();
+            return mphi->getFun();
     }
     return nullptr;
 }

